coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +385,20 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    exclude = False
+    for x in arg_list:
```

Comment:
I saw that :) the problem is that we can have multiple sections, and arg_parser hold only the first section. with `coala -ex sec1, sec2` arg_parser will have only `sec1`